package.json

```diff
@@ -1,7 +1,53 @@
 {
-  "name": "web-common",
-  "dependencies": {
+  "name": "@input-output-hk/purescript-web-common",
+  "version": "1.1.4",
+  "description": "Common web development library for PureScript projects at IO Global",
+  "author": "IO Global",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/input-output-hk/purescript-web-common",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/input-output-hk/purescript-web-common.git"
+  },
+  "bugs": {
+    "url": "https://github.com/input-output-hk/purescript-web-common/issues"
+  },
+  "files": [
+    "static/**/*",
+    "webpack.config.js"
+  ],
+  "scripts": {
+    "test": "spago test --purs-args \"--strict --censor-lib --stash\"",
+    "test:watch": "spago test --purs-args \"--strict --censor-lib --stash\" --watch --clear-screen",
+    "test:js": "jest"
+  },
+  "peerDependencies": {
+    "@popperjs/core": "^2.9.2",
+    "big-integer": "^1.6.50",
+    "chartist": "^0.11.4",
+    "chartist-plugin-axistitle": "^0.0.7",
+    "chartist-plugin-tooltips": "^0.0.17",
+    "clean-webpack-plugin": "^4.0.0",
+    "css-loader": "^6.5.1",
+    "json-bigint": "^1.0.0",
+    "mini-css-extract-plugin": "^2.4.4",
+    "postcss-loader": "^6.2.0"
+  },
+  "devDependencies": {
+    "@popperjs/core": "^2.9.2",
```

Comment:
this is repeated in peerDependencies

Reply:
Yes, as it should be. It's in `devDependencies` here because we don't want it installed when this package is `npm install;d` (It's just used in tests / for local development). But the package that installs web-common is required to install this as a peer dependency. This is all because of the fact that we install this package via `spago` and not `npm` for the purescript dependencies... it's unfortunately rather complicated.